WATOLINK/bci-gaming
# Ryan first level

## ClickUp ticket link

[Create Scene for First Level](https://app.clickup.com/t/8688xk2m6)



## Implementation description
Created the functional components for the first level:
- Platforms
- Fruits
- Enemies
- Background
- Skybox



## Steps to test
1. Go to Scenes/Levels/Level 1 and open the scene
2. Play through the level
3. Check functionality of the level
  a. The cowboy enemies are not functional yet and the bandit enemies behavior is odd, these will be fixed later
4. Review the flow of the level (you generally know where to go)
5. Review the difficulty of the level (may be difficult due to current state of enemies)



## What should reviewers focus on?
Feel free to offer any feedback related to the level design or difficulty. There are several jumps in the level which are designed to be challenging, but these are optional and only lead to extra fruits.


## Checklist
